dev/io.openliberty.threading.virtual.internal/src/io/openliberty/threading/virtual/internal/VirtualThreadOperations.java

```diff
@@ -34,6 +35,13 @@
 @SatisfyingConditionTarget("(&(" + Condition.CONDITION_ID + "=" + JavaInfo.CONDITION_ID + ")(" + JavaInfo.CONDITION_ID + ">=21))")
 public class VirtualThreadOperations implements VirtualThreadOps {
 
+    private VTOverrideService overrideService;
+
+    @Reference
```

Comment:
Default reference cardinality is MANDATORY, meaning that the VirtualThreadOperations service will no longer come up in the absence of the override.  That would be okay if there were an always available built-in instance with lesser service ranking, but I don't see one.  We probably also need a greedy policy option as well in case the stack product feature providing this is added/removed after the server has already come up.  Probably something like this (you will also need imports for some of these classes after adding this)
```suggestion
    @Reference(
        cardinality = ReferenceCardinality.OPTIONAL,
        policy = ReferencePolicy.DYNAMIC,
        policyOption = ReferencePolicyOption.GREEDY)
```

Reply:
Thanks yes was wondering about having a "default" impl - but like it better to have none or make it optional.